providers/smtp/src/main/java/com/sun/mail/smtp/package-info.java

```diff
@@ -226,6 +226,17 @@
  * </TR>
  *
  * <TR>
+ * <TD><A ID="mail.smtp.executor.writetimeout">mail.smtp.executor.writetimeout</A></TD>
+ * <TD>java.util.concurrent.ScheduledExecutorService</TD>
+ * <TD> Provides specific ScheduledExecutorService for mail.smtp.writetimeout option.
+ * The value of mail.smtp.writetimeout shouldn't be a null.
+ * It is highly recommended for provided executor have a setRemoveOnCancelPolicy = true.
```

Comment:
We should use the javadoc link to the SecudeuledThreadPoolExecutor::setRemoveOnCancelPolicy

Reply:
fixed